lapce-data/src/config.rs

```diff
@@ -454,7 +454,7 @@ pub struct EditorConfig {
     #[field_names(
         desc = "Set the cursor blink interval (in milliseconds). Set to 0 to completely disable."
     )]
-    pub blink_interval: u64, // TODO: change to u128 when upgrading config-rs to >0.11
```

Comment:
Why remove the comment? TODO's are important, so much so they're highlighted by default on most terminal text/code editors.

Reply:
This comment is rightfully removed